include/Foundation/NSOperationQueue.h

```diff
@@ -19,25 +19,23 @@
 
 #import <dispatch/dispatch.h>
 #import <Foundation/FoundationExport.h>
+#import <Foundation/NSArray.h>
 #import <Foundation/NSObject.h>
 #import <Foundation/NSOperation.h>
 
 @class NSOperation;
-@class NSArray;
 @class NSString;
 
 enum { NSOperationQueueDefaultMaxConcurrentOperationCount = -1 };
 
-enum { NSOperationQueuePriority_Count = 3 };
-
 FOUNDATION_EXPORT_CLASS
 @interface NSOperationQueue : NSObject
 + (NSOperationQueue*)currentQueue;
 + (NSOperationQueue*)mainQueue;
 - (void)addOperation:(NSOperation*)operation;
-- (void)addOperations:(NSArray*)ops waitUntilFinished:(BOOL)wait;
+- (void)addOperations:(NSArray<NSOperation*>*)ops waitUntilFinished:(BOOL)wait;
 - (void)addOperationWithBlock:(void (^)(void))block;
-@property (readonly, copy) NSArray* operations;
+@property (readonly, copy) NSArray<__kindof NSOperation*>* operations;
```

Comment:
? it's there, it's just hidden under the github cutoff where it doesn't like to show you the whole file